scout/adapter/mongo/case.py

```diff
@@ -995,32 +995,33 @@ def load_case(self, config_data: dict, update: bool = False, keep_actions: bool
                 )
 
         except (IntegrityError, ValueError, ConfigError, KeyError) as error:
-            LOG.warning(error)
-
-        self._load_omics_variants(case_obj, build=genome_build, update=update)
-
-        if existing_case:
-            self.update_case_data_sharing(old_case=existing_case, new_case=case_obj)
-            case_obj["rerun_requested"] = False
-            if case_obj["status"] in ["active", "archived"]:
-                case_obj["status"] = "inactive"
-
-            case_obj["variants_stats"] = self.case_variants_count(
-                case_id=case_obj["_id"],
-                institute_id=institute_obj["_id"],
-                force_update_case=True,
-            )
+            LOG.exception(error)
+            raise error
+        else:
+            self._load_omics_variants(case_obj, build=genome_build, update=update)
```

Comment:
My feeling was we don't really need to go ahead with more variants if some have already failed, but the case should still be updated to show it no longer has variants, when it was touched, and do our best to preserve info as much as we can if that was requested. But still reraise again to get a proper non-0 exit status for downstream apps to notice.

Reply:
Sure, I would agree to that - better to catch anything that may be there, but perhaps a little less clear that this happens only if the rest works with an "else".